### PR TITLE
Interactive separators

### LINKED ISSUE #1184: Separators

### PR DESCRIPTION
**Pull request by kdzwinel**
Closes #1184

screen shot 2017-07-27 at 12 18 04

### CODE REVIEW

File: src/docs/_sass/_docs-active-block.scss [hunk 1]
```diff
@@ -13,6 +13,8 @@ $docsActiveBlockTransparent: rgba(0, 0, 0, 0);
 
   &--dark .docs-active-block__component {
     background-color: $grayPrimary;
+    background-size: 41px 40px;
+    background-image: radial-gradient(circle, $navybluePrimary 1px, $docsActiveBlockTransparent 1px);
```
Comment (kdzwinel) [inline src/docs/_sass/_docs-active-block.scss]: without the dots this background was very boring 🤡

File: src/docs/components/DocsActiveBlock.jsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ class DocsActiveBlock extends Component {
         .reduce((result, key) => {
           result[key] = componentProps[key];
           return result;
-        }, {});
+        }, {key: 'component'});
```
Comment (kdzwinel) [inline src/docs/components/DocsActiveBlock.jsx]: To stop React from complaining I have to add a `key` to component, contentBefore, contentAfter

File: src/docs/components/DocsActiveBlock.jsx [hunk 3]
```diff
@@ -77,17 +80,35 @@ class DocsActiveBlock extends Component {
       'docs-active-block--dark': this.state.changeBackground === 'dark'
     });
 
-    component = [this.props.contentBefore, component, this.props.contentAfter];
+    output = [];
 
-    if (this.props.wrapper) {
-      component = React.cloneElement(this.props.wrapper, {}, component);
+    if (contentBefore) {
+      if (React.isValidElement(contentBefore)) {
```
Comment (kdzwinel) [inline src/docs/components/DocsActiveBlock.jsx]: if contentBefore is e.g. text, don't try to clone it

File: src/docs/components/DocsActiveBlock.jsx [hunk 4]
```diff
@@ -77,17 +80,35 @@ class DocsActiveBlock extends Component {
       'docs-active-block--dark': this.state.changeBackground === 'dark'
     });
 
-    component = [this.props.contentBefore, component, this.props.contentAfter];
+    output = [];
```
Comment (matikrk) [inline src/docs/components/DocsActiveBlock.jsx]: you could put 'let' here, not top of function

Comment (kdzwinel) [inline src/docs/components/DocsActiveBlock.jsx]: true, fixed

File: src/components/dropdowns/pages/dropdowns.jsx [hunk 5]
```diff
@@ -20,20 +20,20 @@ const Dropdowns = () =>
     <DocsBlock info='Open' multiContent={[
       <div style={{height: '120px'}}>
         <ContrastBox>
-          <Dropdown {...defaultProps} isOpened={true} fullWidth={false}/>
+          <Dropdown {...defaultProps} openOnStart={true} fullWidth={false}/>
```
Comment (kdzwinel) [inline src/components/dropdowns/pages/dropdowns.jsx]: fixing demo after previous PR

**Pull request merged**